sickbeard/properFinder.py

```diff
@@ -158,7 +158,7 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
             # if they haven't been added by a different provider than add the proper to the list
             for proper in cur_propers:
                 guess = guessit(proper.name)
-                if not guess.get('proper_tags'):
+                if not guess.get('proper_count'):
```

Comment:
This can be completely removed/replaced now, since we know if it is a proper from the NameParser.